tcf_website/tests/test_fetch_enrollment.py

```diff
@@ -0,0 +1,84 @@
+"""Tests for fetch_enrollment management command."""
+
+from unittest.mock import patch
+from django.test import TestCase
+from tcf_website.models import SectionEnrollment
+from tcf_website.management.commands.fetch_enrollment import fetch_section_data
+from .test_utils import setup
+
+
+class FetchEnrollmentTestCase(TestCase):
+    """Tests for the fetch_enrollment command."""
+
+    def setUp(self):
+        """Set up test data."""
+        setup(self)
+        # pylint: disable=no-member
+        self.section = self.section_course
+
+    @patch("tcf_website.management.commands.fetch_enrollment.session.get")
+    def test_fetch_enrollment_success(self, mock_get):
+        """Test successful enrollment fetch."""
```

Comment:
is the rest of the codebase using unittest.mock? This flows a lot better than some of the older tests.

Reply:
I don't think so, but future tests should probably use this.